docs/REST API/rest-api.md

```diff
@@ -6176,6 +6187,219 @@ Content-Length: <length>
 Body: <blob>
 ```
 
+### List software (setup experience)
+
+_Available in Fleet Premium_
+
+List software that can or will be automatically installed during macOS setup. If `included_during_setup` is `true` it will be installed during setup.
```

Comment:
@marko-lisica this says `included_during_setup`, but the response below has `install_during_setup`. Which of these is correct?

Reply:
@rachaelshaw Thanks for catching this. I just made a PR `install_during_setup` is correct.